src/SprykerEco/Zed/UnzerRestApi/Communication/Plugin/CheckoutRestApi/UnzerCheckoutDataExpanderPlugin.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\UnzerRestApi\Communication\Plugin\CheckoutRestApi;
+
+use Generated\Shared\Transfer\RestCheckoutDataTransfer;
+use Generated\Shared\Transfer\RestCheckoutRequestAttributesTransfer;
+use Spryker\Zed\CheckoutRestApiExtension\Dependency\Plugin\CheckoutDataExpanderPluginInterface;
+use Spryker\Zed\Kernel\Communication\AbstractPlugin;
+
+/**
+ * @method \SprykerEco\Zed\UnzerRestApi\Business\UnzerRestApiFacadeInterface getFacade()
+ * @method \SprykerEco\Zed\UnzerRestApi\UnzerRestApiConfig getConfig()
+ */
+class UnzerCheckoutDataExpanderPlugin extends AbstractPlugin implements CheckoutDataExpanderPluginInterface
+{
+    /**
+     * {@inheritDoc}
+     * - Requires `RestCheckoutDataTransfer.quote.store.name` transfer property to be set.
+     * - Expands `RestCheckoutDataTransfer.quote` transfer property with `UnzerCredentialsTransfer` according to added items.
+     * - Does nothing if quote has no items.
+     *
+     * @api
+     *
+     * @param \Generated\Shared\Transfer\RestCheckoutDataTransfer $restCheckoutDataTransfer
+     * @param \Generated\Shared\Transfer\RestCheckoutRequestAttributesTransfer $restCheckoutRequestAttributesTransfer
+     *
+     * @return \Generated\Shared\Transfer\RestCheckoutDataTransfer
+     */
+    public function expandCheckoutData(
```

Comment:
call facade method in the same way `expandCheckoutData`
and move code from plugin to business model
leave only facade call here